spec/factories/tasks.rb

```diff
@@ -68,7 +68,14 @@
 
     trait :random_completed_in_range do
```

Comment:
Never use traits with randomness!! This will make tests non-deterministic and may cause situations when one time tests will pass and another will not. If you need different traits just make two separate ones, one with data and one with nil.

Reply:
це тільки для сідів, в тестах це не використовується

Reply:
Are you sure no one will ever use it in tests? If you need randomness in seeds than move it to seeds, write something like `[true, false].sample ? create(:task, :trait_one) : create(:task, :trait_two)` inside seeds